third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/ScanOpToLLVM.cpp

```diff
@@ -513,6 +544,10 @@ ScanOpConversion::emitFastScan(triton::ScanOp op, triton::ScanOpAdaptor adaptor,
   };
 
   SmallVector<Value> results(op.getNumOperands());
+  if (op.getReverse()) {
+      srcValues = flipSrcValues(loc, op, rewriter, srcValues, iWarpSize);
```

Comment:
It should be 2 space indented to be consistent with our style. I thought `pre-commit` is able to format it automatically.

Reply:
I think we might be having formater issues after moving code to third_party (@zahimoud do you know?)

Reply:
third_party/ is in the exclude directory for clang-format. I'll run it manually.

Reply:
ah good catch, we should include at least third_party/nvidia, we should fix it in another PR (anybody wants to do it?)

Reply:
I'll do it